packages/time-select/src/time-select.vue

```diff
@@ -144,7 +144,7 @@ export default defineComponent({
   emits: ['change', 'blur', 'focus', 'update:modelValue'],
   setup(props) {
     // data
-    const value = ref(props.modelValue)
+    const value = computed(() => props.modelValue)
```

Comment:
I guess the `value` is unnecessary, we can do `:model-value="modelValue"` in el-select

Reply:
Yep, that's the essential idea of not introducing more variables. The less the better.